sched_slack_bot/controller.py

```diff
@@ -91,7 +93,19 @@ def start(self) -> None:
 
         self._start_all_saved_schedules()
         self._register_listeners()
-        self._app.start(port=port)
+
+        app_handler = SlackRequestHandler(self._app)
+        api = FastAPI()
```

Comment:
not completely against this, but somehow i think I would prefer starting FastAPI outside of the controller (e.g. in the bin file?), it's not related to the bot just to the kubernetes health check, right?

Reply:
Well the FastAPI object routes incoming http requests to the healthcheck _and_ to the bot. See the two usages of `api` annotations just below

Reply:
yes i know, but we even added the full fastApi dependency just to do the extra health route. im also okay with it like this, but seems like a lot just to expose a health check route, and imho it just doesn't belong in the SchedSlackBotController (since it doesn't need a health check)